docs/source/topics/middleware.rst

```diff
@@ -271,15 +271,15 @@ Integrating with AWS Lambda Powertools
 --------------------------------------
 
 `AWS Lambda Powertools
-<https://awslabs.github.io/aws-lambda-powertools-python/latest/>`__ is a suite of
+<https://docs.powertools.aws.dev/lambda/python/latest/>`__ is a suite of
```

Comment:
the prcheck link checker complained (failed) about these powertools links.  The redirect seemed ok to me in my browser, but, updated them.